### DISCUSSION TITLE
Hi, how to install in EMMC Orange 5 pi B

### DISCUSSION THREAD

**Discussion by mlaureano** [Joshua-Riek/ubuntu-rockchip#188] [General]
First of all congratulations on the initiative. The distro via sd card is very responsive and fast, but I want to install it on the EMMC of the Orange PI 5B and I can't even "see" the internal disk. I've looked in other documentation and I couldn't find a solution, any suggestions?

**Comment by Joshua-Riek** [comment]
I have a guide here on how to install Ubuntu onto your eMMC:
https://github.com/Joshua-Riek/ubuntu-rockchip/wiki/Orange-Pi-5#install-ubuntu-onto-your-emmc-from-linux

**Comment by nobara256** [comment]
There should be an `mmcblk0` entry in `lsblk`.

**Comment by mlaureano** [comment]
1. I tried to boot with the ubuntu image, but it didn't recognize the eMMC.

2. I booted with the OPI OS image and saved the Ubuntu image in the eMMC and it didn't boot correctly (it stayed at the administrative command prompt due to an initialization error).

**Reply by Joshua-Riek** [reply]
Make sure to download the image for the Orange Pi 5B. The eMMC can not be recognized on the regular Orange Pi 5 image.

**Comment by Z-Man23** [comment]
Booted from SD Card, looks good.

Downloaded "ubuntu-22.04.3-preinstalled-desktop-arm64-orangepi-5b.img.xz"

Opened terminal and copy pasted from the linked page above (edited slightly to account for the 22.04.3 rather than 22.04.2) and got the following error:

"~$ xz -dc ubuntu-22.04.3-preinstalled-desktop-arm64-orangepi-5b.img.xz | sudo dd of=/dev/mmcblk0 bs=4k
sync
xz: ubuntu-22.04.3-preinstalled-desktop-arm64-orangepi-5b.img.xz: No such file or directory
0+0 records in
0+0 records out
0 bytes copied, 0.0125187 s, 0.0 kB/s"

Is there a specific place I should be saving the file to?  I just clicked "download" This is my first time working with Ubuntu.

   Any help would be appreciated.

**Reply by wmport** [reply]
check in which directory the terminal is opened and the name of the downloaded file
